nadlogar/problems/forms.py

```diff
@@ -47,11 +47,6 @@ def clean(self):
                         ] = "Da ne bi prišlo do izgube podatkov, mora biti ob izbiri obstoječega besedila to polje prazno."
             else:
                 del self.errors["text"]
-                for field_name in ["instruction", "solution"]:
```

Comment:
Če prav razumem, smo to izbrisali, ker bosta od zdaj naprej instruction in solution že po defaultu vsebovala neko besedilo.

Reply:
Tako ja. Pomen obrazca se je spremenil. Prej je moral biti bodisi izbran tekst bodisi vpisani obe polji. Zdaj je možnost, da je vse troje prazno in se izbere privzeto besedilo.